### ISSUE TITLE
Adjusting of icon size for better grid alignment?

### ISSUE THREAD

**Issue by sprite-1** [spheras/desktopfolder#230]
It slightly bugs me that the grid isn't horizontally aligned  as seen below.

![screenshot from 2018-12-19 14 08 25](https://user-images.githubusercontent.com/33685433/50202163-0a0c4780-0398-11e9-84d2-2f29c881705c.png)

I think having a feature to allow icon + text size slider adjustments would solve this problem on a per-person basis with differing screen sizes. :thinking: 

**Edit:**

Or at the very least, allow us to adjust the spacing of the grids manually.

**Comment by sprite-1** [comment]
Heya @spheras this is what I'm talking about

![image](https://user-images.githubusercontent.com/33685433/50212272-94af6f80-03b5-11e9-8652-76779e527ac2.png)

**Comment by spheras** [comment]
Ah, Ok, I see. Yes, you could adjust this manually adding extra spaces between the icons (in the last version).  So, If I'm not wrong your idea is to adjust automatically the icon size to fit exactly the grid in the desktop, isn't it?  I think it would be better to adjust the cell of the grid size, instead of the icon size (now is not possible, and in the future, the size will have a default but could be increased manually. 

The situation would be more complex to solve in the case of multimonitors, which at the end of the day, is like a bigger panel... and then how could be calculated the size? taking into account the whole panel? only the main monitor? (probably different monitors size)

**Comment by spheras** [comment]
No, that version is not released yet. The new version we are working now is not released and will have huge improvements... hope during January/February will be released (but not sure).  Meanwhile, you can compile by yourself and check (if you want).

Yes, that could be a good idea, only take into account adjust if there is only one monitor.  I'm tagging this as a feature request, and I will consider. Thanks!

**Comment by sprite-1** [comment]
I'll have to see if I can figure out how to compile once I'm free, thank you for the consideration!
